docs/registry.md

```diff
@@ -464,6 +464,11 @@ By default, your markdown file __inherits__ much of its metadata from the topic
 **`resourceType`**:
     this will override the global resource Type as defined in the registry level configuration. Valid resource types are `Components, Documentation, Repositories, Self-Service Tools`
 
+**`Tags`**:
+Tag has __very high__ search priority in dev hub. Tags are an array of relevant keywords that belong to your content. Belong example includes an acceptable tags field format for devhub 
```

Comment:
Tags are a way to increase the 'discoverability' of your content via keywords. These keywords score higher in a search in comparison to the contents title and content body.